extensions/ql-vscode/src/packaging/suite-instruction.ts

```diff
@@ -5,8 +5,8 @@ export interface SuiteInstruction {
   qlpack?: string;
   query?: string;
   queries?: string;
-  include?: Record<string, string[]>;
-  exclude?: Record<string, string[]>;
+  include?: Record<string, string[]> | Record<string, string>;
+  exclude?: Record<string, string[]> | Record<string, string>;
```

Comment:
Should this be something like this because it's possible to mix `string` and `string[]` in the same `include`/`exclude` property?

```suggestion
  include?: Record<string, string | string[]>;
  exclude?: Record<string, string | string[]>;
```

Reply:
Yes. Thanks. That's a better option. I'll make that change.